apps/meteor/ee/app/livechat-enterprise/server/hooks/beforeRoutingChat.js

```diff
@@ -59,7 +59,11 @@ const beforeRouteChat = async (inquiry, agent) => {
 	saveQueueInquiry(inquiry);
 
 	if (settings.get('Omnichannel_calculate_dispatch_service_queue_statistics')) {
-		const [inq] = await LivechatInquiry.getCurrentSortedQueueAsync({ _id, department });
+		const [inq] = await LivechatInquiry.getCurrentSortedQueueAsync({
+			_id,
```

Comment:
looks like `getCurrentSortedQueueAsync` doesn't accept an `_id` param 🤔  do you mean `inquiryId` instead?